movies/3.api-doc.md

```diff
@@ -0,0 +1,89 @@
+# The API documentation
+
+## Story
+
+As a client developer, I want to find human-readable documenation of the API.
+As a client, I want to find a machine-readable description of the API capabilities.  
+
+## API behaviour
+
+```http
+GET https://hydra-movies.herokuapp.com/
+```
+
+Response headers (excerpt):
+
+```http
+Link: </doc>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
+```
+
+```http
+GET https://hydra-movies.herokuapp.com/doc
+```
+
+Response body (excerpt):
+
+```json
+{
+  "@context": "/context.jsonld",
```

Comment:
Changed to absolute context. Unfortunately, if wee want to keep relative URIs in the representation we need `@base` too.

```suggestion
  "@context": [
    "https://hydra-movies.herokuapp.com/context.jsonld",
    {
      "@base": "https://hydra-movies.herokuapp.com/"
    }
  ]
```

Reply:
Why do you duplicate the whole context here, instead of using a (absolute) link?

I always wonder, do we really need `@base`? I thought the client could assume the API Origin as base if none given explicitly ?

Reply:
> I thought the client could assume the API Origin as base if none given explicitly?

It's same as with the relative context discussion we had. Just looking at raw triples you don't know what the base is.

> If we need to set @base we should move it to context.jsonld, shouldn't we?

Actually that's a good idea. You're right, it should work 👍